src/audio/google/google_rtc_audio_processing.c

```diff
@@ -704,30 +704,6 @@ static int google_rtc_audio_processing_copy(struct comp_dev *dev)
 }
 
 #if CONFIG_IPC_MAJOR_4
-static int google_rtc_audio_processing_set_large_config(struct comp_dev *dev,
-							uint32_t param_id,
-							bool first_block,
-							bool last_block,
-							uint32_t data_offset,
-							const char *data)
-{
-	struct google_rtc_audio_processing_comp_data *cd = comp_get_drvdata(dev);
-	size_t bcfg_size, ext_size;
-
-	switch (param_id) {
-	case IPC4_AEC_SET_EXT_FMT:
-		bcfg_size = sizeof(struct ipc4_base_module_cfg);
-		ext_size = sizeof(cd->config) - bcfg_size;
-		memcpy_s((void *)&cd->config + bcfg_size, ext_size, data, ext_size);
```

Comment:
so frame format is fixed, and can't be changed after module_init, if there is new stream with RTC, then module_init will be called again.

Reply:
no it wont. the module is initialized only once